### PR TITLE
Return NotFoundError if kid not found

### PR DESCRIPTION
**Pull request by patrickxb**
Regardless of how it happened that LoadV2() returned `nil, nil, X, nil`, I think this is better than crashing.

### CODE REVIEW

File: go/libkb/upak_loader.go [hunk 1]
```diff
@@ -500,11 +500,12 @@ func (u *CachedUPAKLoader) LoadKeyV2(ctx context.Context, uid keybase1.UID, kid
...
 		if key != nil {
 			u.G().VDL.CLogf(ctx, VLog0, "- found kid in UPAK: %v", *ret)
-			break
+			return ret, key, linkMap, nil
 		}
 		ret = nil
 	}
-	return ret, key, linkMap, nil
+
+	return ret, key, linkMap, NotFoundError{}
```
Comment (maxtaco) [inline go/libkb/upak_loader.go]: can this just be `nil, nil, nil, NotFoundError{}` ? Always tricky to return data and an error, best to avoid it.

**Comment by maxtaco** [comment]
What do you think we should do for the other issue?  I'm worried this team is now broken

**Comment by maxtaco** [comment]
can you hack your keybase client to run `LoadKeysV2()` on that user+kid combo to see if it works?

**Comment by patrickxb** [comment]
I'm waiting to see if `db nuke` solved it.  Haven't heard back.

I'll try LoadKeysV2() with that and see what happens.

**Comment by patrickxb** [comment]
(hacked client `LoadKeysV2` worked fine for uid:kid pair in question)

**Pull request merged**